verify/inconsistency/stats.go

```diff
@@ -33,14 +35,20 @@ func (s *RowStats) String() string {
 // reportRunningSummary reports the number of total rows and errors seen
 // during the execution of verify.
 func reportRunningSummary(l zerolog.Logger, s RowStats, m string) {
+	if s.NumConditionalSuccess > 0 {
+		m = fmt.Sprintf("%s - please check all warnings and errors for column mismatches to determine success", m)
```

Comment:
Something like this?

```suggestion
		m = fmt.Sprintf("%s - check all warnings and errors to determine whether column mismatches are breaking", m)
```

Reply:
Done!